ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -1430,31 +1426,30 @@ def test_delete_configuration(self, mock_delete_virtual_disk,
 
     @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
                        autospec=True)
-    @mock.patch.object(drac_raid, 'list_virtual_disks', autospec=True)
+    @mock.patch.object(drac_raid, 'list_raid_controllers', autospec=True)
     @mock.patch.object(drac_job, 'validate_job_queue', spec_set=True,
                        autospec=True)
     @mock.patch.object(drac_raid, 'commit_config', spec_set=True,
                        autospec=True)
-    @mock.patch.object(drac_raid, 'delete_virtual_disk', spec_set=True,
+    @mock.patch.object(drac_raid, 'reset_raid_config', spec_set=True,
                        autospec=True)
-    def test_delete_configuration_no_change(self, mock_delete_virtual_disk,
+    def test_delete_configuration_no_change(self, mock_reset_raid_config,
                                             mock_commit_config,
                                             mock_validate_job_queue,
-                                            mock_list_virtual_disks,
+                                            mock_list_raid_controllers,
                                             mock_get_drac_client):
         mock_client = mock.Mock()
         mock_get_drac_client.return_value = mock_client
-        mock_list_virtual_disks.return_value = []
-        mock_delete_virtual_disk.return_value = {
+        mock_list_raid_controllers.return_value = []
+        mock_reset_raid_config.return_value = {
             'is_reboot_required': 'optional',
```

Comment:
I think 'optional' on the line above needs to be replaced with constants.RebootRequired().optional